### ISSUE TITLE
Thoughts on /dev/null owned by tss (current implementation makes ansible fail on this task)

### ISSUE THREAD

**Issue by kris9854** [ansible-lockdown/RHEL9-CIS#24]
**Question**
As im not sure if it is a bug i place it in the quation section.

Maybe also implement a skip for the tss user to fix failure:
failed: [rhel9-hardning] (item=tss: /dev/null) => {"ansible_loop_var": "item", "changed": false, "gid": 0, "group": "root", "item": {"dir": "/dev/null", "gecos": "Account used for TPM access", "gid": 59, "id": "tss", "password": "x", "shell": "/sbin/nologin", "uid": 59}, "mode": "0666", "msg": "/dev/null already exists as a file", "owner": "root", "path": "/dev/null", "secontext": "system_u:object_r:null_device_t:s0", "size": 0, "state": "file", "uid": 0
I cant really see why tss should be owning /dev/null- following cis benchmark isn't it meant for the /dev/nul

**Additional Notes**
Before change the task looks like:
````yaml
- name: "6.2.10 | PATCH | Ensure users own their home directories"
  file:
      path: "{{ item.dir }}"
      owner: "{{ item.id }}"
      state: directory
  with_items: "{{ rhel9cis_passwd }}"
  loop_control:
      label: "{{ rhel9cis_passwd_label }}"
  when:
      - min_int_uid | int <= item.uid
      - rhel9cis_rule_6_2_10
  tags:
      - skip_ansible_lint  # settings found on 6_2_7
      - level1-server
      - level1-workstation
      - automated
      - patch
      - users
      - rule_6.2.10
````

**Possible Solution**:

```yaml
- name: "6.2.10 | PATCH | Ensure users own their home directories"
  file:
      path: "{{ item.dir }}"
      owner: "{{ item.id }}"
      state: directory
  with_items: "{{ rhel9cis_passwd }}"
  loop_control:
      label: "{{ rhel9cis_passwd_label }}"
  when:
      - min_int_uid | int <= item.uid
      - (item.id != 'tss' and item.dir != '/dev/null') #Added for not implementing owner of /dev/null on tss user
      - rhel9cis_rule_6_2_10
  tags:
      - skip_ansible_lint  # settings found on 6_2_7
      - level1-server
      - level1-workstation
      - automated
      - patch
      - users
      - rule_6.2.10
````

**Comment by uk-bolly** [comment]
hi @kris9854 

Thank you for the feedback and the suggested solution. I dont believe this has yet been raised or covered off by outstanding PRs. I will investigate further on this.

Many thanks once again

uk-bolly

**Comment by uk-bolly** [comment]
hi @kris9854 

Looking through this was closed by #28 
I will close this Issue, please feel free to reopen if this is not the case.

many thanks

uk-bolly
